app/javascript/stylesheets/sponsors-tile.css

```diff
@@ -15,7 +15,7 @@
     sm:flex-row
     justify-center
     items-center
-    max-w-sm
```

Comment:
I changed the layouts just a tad to make it look better with three logos instead of two.